Wikipedia/Code/PageHistoryCountsView.xib

```diff
@@ -83,23 +83,38 @@
                     </constraints>
                 </view>
                 <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="ijp-FN-hau" userLabel="Filter Counts Container View">
-                    <rect key="frame" x="50" y="180.33333333333334" width="702" height="279.66666666666663"/>
+                    <rect key="frame" x="50" y="180.33333333333334" width="702" height="279.33333333333326"/>
+                </view>
+                <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="JxZ-iv-nug" userLabel="Separator View Bottom">
+                    <rect key="frame" x="0.0" y="459.66666666666669" width="802" height="0.33333333333331439"/>
+                    <constraints>
+                        <constraint firstAttribute="height" constant="0.5" id="d2h-ka-oga"/>
+                    </constraints>
+                    <userDefinedRuntimeAttributes>
+                        <userDefinedRuntimeAttribute type="color" keyPath="borderColor">
+                            <color key="value" systemColor="systemPinkColor"/>
```

Comment:
I think this `systemPinkColor` attribute may have been test code, can you remove it?

<img width="1211" alt="Screenshot 2025-07-02 at 4 53 46 PM" src="https://github.com/user-attachments/assets/54ca1a00-65ed-4818-bbe9-7a2fe327971c" />